### DISCUSSION TITLE
✅ Product Ajax Search in Elementor PRO: "When?" It's 2024

### DISCUSSION THREAD

**Discussion by DanielCyran** [elementor/elementor#24931] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

I am glad that Elementor PRO is constantly being developed. However, it has been missing an Ajax product search for many years, which is surprising given that every online store owner needs such an Ajax search.

The introduction of such a search engine will make the stores of people with Elementor PRO less cluttered. Additionally, Elementor PRO will become a fully functional tool for creating online stores, as the current lack of a basic function such as Ajax search causes people to search for and install unnecessary plugins that slow down stores.

### Describe the solution you'd like.

I kindly ask you to take this matter seriously and introduce an Ajax product search to Elementor PRO. Its absence has been felt for many years.

**See how many YouTube videos are related to AJAX search. This shows how high the demand for this feature is and how many users are polluting their websites due to the lack of such a feature in Elementor Pro:**
https://www.youtube.com/results?search_query=ajax+product+search+elementor

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by DanielCyran** [comment]
Can the Elementor PRO team provide information about the introduction of an AJAX product search?

**Comment by nicholaszein** [comment]
Hey @DanielCyran!

We have great news! 🙌 

### 📢 We're happy to announce that this feature was released with Elementor Pro v3.23! 🥳 

You can now use the [new Search widget with Live Results](https://elementor.com/help/search-widget/), with the option to choose your query as well!

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
